osutil/user/user.go

```diff
@@ -0,0 +1,60 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build !snapdusergo
```

Comment:
do we need a new unit tests variant in GH, or changes to the current ones? cc @bboozzoo